scipy/stats/tests/test_morestats.py

```diff
@@ -1575,23 +1575,25 @@ def optimizer(fun, args):
         assert bounds[0] < lmbda < bounds[1]
 
     def test_bounded_optimizer_against_unbounded_optimizer(self):
-        # Test whether setting bounds on optimizer excludes solution from unbounded
-        # optimizer.
+        # Test whether setting bounds on optimizer excludes solution from
+        # unbounded optimizer.
 
         # Get unbounded solution.
         _, lmbda = stats.boxcox(_boxcox_data, lmbda=None)
 
-        # Set bounds around solution.
-        bounds = (lmbda - 1, lmbda + 1)
```

Comment:
This does _not_ exclude the solution from the unbounded optimizer, hence change.